web/modules/custom/server_style_guide/templates/server-style-guide-person.html.twig

```diff
@@ -0,0 +1,24 @@
+<div class="w-80 h-96 bg-white rounded-lg shadow flex-col justify-start items-start inline-flex">
```

Comment:
Thanks. This is not aligned with how it's structured on DS (Drupal-starter). You can fetch the latest, and re-work it. There shouldn't be one huge twig file.

Please remove the GitHub actions reviews, they are just adding noise